### PR TITLE
Include positioning data package summary and description translations

### PR DESCRIPTION
**Pull request by jpetrell**

### CODE REVIEW

File: rpm/mlsdb-data.spec [hunk 1]
```diff
@@ -5,6 +5,9 @@ Summary: Geoinformation data from Mozilla Location Services Geoclue provider
 URL: https://github.com/sailfishos/mlsdb-data/
 License: Public Domain
 Source0: %{name}-%{version}.tar.gz
+BuildRequires:  pkgconfig(Qt5Core)
```
Comment (inzanity) [inline rpm/mlsdb-data.spec]: Is there a reason for this?

File: rpm/mlsdb-data.spec [hunk 2]
```diff
@@ -46,6 +49,10 @@ Moldova, Poland, Romania, Slovakia and Ukraine.
 %prep
 %setup -q -n %{name}-%{version}
 
+%build
+%qmake5 translations
```
Comment (inzanity) [inline rpm/mlsdb-data.spec]: Would it be simpler to just call lupdate and copy the generated file in place?

File: translations/StoreDescription.qml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Item {
+    // providing dummy translations for app descriptions shown on Store
+    function qsTrIdString() {
+
+        // Northern Europe
+
+        //% "Mozilla Location Services offline positioning data for Northern Europe speeds up getting position lock for your device in the region."
+        QT_TRID_NOOP("mlsdb-data-la-store_northern_europe_summary")
+
+        //% "Northern Europe package contains positioning data for countries Denmark, Estonia, Finland, Iceland, Ireland, Latvia, Lithuania, Norway, Sweden and United Kingdom.\n"
```
Comment (inzanity) [inline translations/StoreDescription.qml]: I'd leave the "countries" off.

File: rpm/mlsdb-data.spec (hunk 1 above)
Comment (jpetrell) [reply rpm/mlsdb-data.spec]: Nope. Removed. 🙂

File: rpm/mlsdb-data.spec (hunk 2 above)
Comment (jpetrell) [reply rpm/mlsdb-data.spec]: Duh. It would be. Fixed.

File: translations/StoreDescription.qml (hunk 3 above)
Comment (jpetrell) [reply translations/StoreDescription.qml]: True. Removed.

**Review by inzanity — APPROVED** [review]
LGTM

File: translations/StoreDescription.qml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        QT_TRID_NOOP("mlsdb-data-la-store_southern_europe_summary")
+
+        //% "Southern Europe package contains positioning data for Albania, Andorra, Bosnia and Herzegovina, Croatia, Greece, Italy, Macedonia, Malta, Montenegro, Portugal, San Marino, Serbia, Slovenia and Spain, but not including Holy See (Vatican).\n"
+        //% "\n"
+        //% "Positioning using the data does not require network access. Positioning data contains mobile phone tower information for the region. The data is based on open Mozilla Location Service cell network information published under public domain in https://location.services.mozilla.com"
+        QT_TRID_NOOP("mlsdb-data-la-store_southern_europe_description")
+
+        // Eastern Europe
+
+        //% "Mozilla Location Services offline positioning data for Southern Europe speeds up getting position lock for your device in the region."
```
Comment (sledges) [inline translations/StoreDescription.qml:39]: Eastern

File: translations/StoreDescription.qml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        //% "\n"
+        //% "Positioning using the data does not require network access. Positioning data contains mobile phone tower information for the region. The data is based on open Mozilla Location Service cell network information published under public domain in https://location.services.mozilla.com"
+        QT_TRID_NOOP("mlsdb-data-la-store_western_europe_description")
+
+        // Southern Europe
+
+        //% "Mozilla Location Services offline positioning data for Southern Europe speeds up getting position lock for your device in the region."
+        QT_TRID_NOOP("mlsdb-data-la-store_southern_europe_summary")
+
+        //% "Southern Europe package contains positioning data for Albania, Andorra, Bosnia and Herzegovina, Croatia, Greece, Italy, Macedonia, Malta, Montenegro, Portugal, San Marino, Serbia, Slovenia and Spain, but not including Holy See (Vatican).\n"
```
Comment (sledges) [inline translations/StoreDescription.qml:32]: the Holy See

**Pull request merged**